x/epochs/keeper/epoch_test.go

```diff
@@ -6,6 +6,60 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 )
 
+func (suite *KeeperTestSuite) TestAddEpochInfo() {
+	defaultIdentifier := "default_add_epoch_info_id"
+	defaultDuration := time.Hour
+	startBlockHeight := int64(100)
+	startBlockTime := time.Unix(1656907200, 0).UTC()
+	tests := map[string]struct {
+		addedEpochInfo types.EpochInfo
+		expErr         bool
+		expEpochInfo   types.EpochInfo
+	}{
+		"simple_add": {addedEpochInfo: types.EpochInfo{
```

Comment:
I think formatting might be off here. Should this be on the next line?

Reply:
yeah whole tests fields' formatting would be better fixed before merging!